man/crypttab.xml

```diff
@@ -172,12 +172,11 @@
       <varlistentry>
         <term><option>header=</option></term>
 
-        <listitem><para>Use a detached (separated) metadata device or
-        file where the LUKS header is stored. This option is only
-        relevant for LUKS devices. See
+        <listitem><para>Use a detached (separated) metadata device or file
+        where the encrypted header containing the master key(s) is stored. This
```

Comment:
The header is not really encrypted though, part of it is plaintext. Just call it "header".